src/components/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,11 @@
+import './Navigation.css';
+
+const Navigation = () => (
+  <nav className="navigation">
+    <a className="navigation__link" href="#about">About</a>
+    <a className="navigation__link" href="#services">Services</a>
+    <a className="navigation__link" href="#contact">Contact</a>
+  </nav>
```

Comment:
```suggestion
  <nav className="navigation">
    <a className="navigation__link" href="#about">About</a>

    <a className="navigation__link" href="#services">Services</a>

    <a className="navigation__link" href="#contact">Contact</a>
  </nav>
```
FYI: it's a good practice to add a blank line between **sibling** elements to increase readability - take it into account in the future